symbols.js

```diff
@@ -413,6 +426,63 @@ function search(searchTerm) {
     });
 }
 
+function addSymbol() {
+    symbols.unshift({
+        glyph: "",
+        name: "",
+        // For now: no display and search terms
+    });
+    // Better to just add the one div for the symbol
+    // For now this is fast enough.
+    renderSymbols();
+    editSymbol(document.getElementsByClassName("symbol")[0], "glyph");
+}
+
+function editSymbol(elem, classname) {
```

Comment:
Indentation of this function is a little off, and this `console.log` should be removed before merging.

Reply:
This (and other) console.log removed in 914415c9dd6387c7983bcf6ddd23508b77b5c2fe